app/views/shared/_assignment_stats.html.erb

```diff
@@ -1,40 +1,48 @@
+  
     <div class="display-flex flex-column tablet:flex-row tablet:flex-align-center">
       <div class="tablet:flex-fill">
-        <div class="display-flex flex-align-center margin-bottom-2 tablet:margin-bottom-0">
-          <div class="margin-right-2">
-            <span class="font-sans-3xl text-primary text-bold"><%= @assigned_submissions.count %></span>
-          </div>
-          <div>
-            <h3 class="usa-summary-box__heading text-primary margin-bottom-05" id="summary-box-key-information">
-              <%= heading %>
-            </h3>
-            <p class="usa-summary-box__text text-primary-darker text-bold">
-              Evaluations due by <%= @phase.evaluation_form&.closing_date&.strftime('%m/%d/%Y') || "N/A" %>
-            </p>
-          </div>
-        </div>
+        <ul class="add-list-reset">
+          <li class="margin-bottom-0.5 display-flex">
+              <span class="text-bold text-black" style="width: 6rem; text-align: right"><%= @assigned_submissions.count %></span>
+              <span class="margin-left-2">Assigned submissions</span>
+          </li>
+          <% if local_assigns[:show_all_stats] %>
+            <li class="margin-bottom-0.5 display-flex">
+              <span class="text-bold text-black" style="width: 6rem; text-align: right"><%= @recused_submissions.count %></span>
+              <span class="margin-left-2">Recused submissions</span>
+            </li>
+            <li class="margin-bottom-0.5 display-flex">
+              <span class="text-bold text-black" style="width: 6rem; text-align: right"><%= @unassigned_submissions.count %></span>
+              <span class="margin-left-2">Unassigned submissions</span>
+            </li>
+          <% end %>
+          <li class="display-flex">
+              <span class="text-bold text-black" style="width: 6rem; text-align: right"><%= @phase.evaluation_form&.closing_date&.strftime('%m/%d/%Y') || "N/A" %></span>
+              <span class="margin-left-2">Evaluation due date</span>
+          </li>
+        </ul>
       </div>
 
-      <div class="border-top border-base-dark margin-y-2 tablet:display-none"></div>
-
-      <div class="display-flex flex-row tablet:margin-left-auto">
-        <div class="text-center margin-right-1 margin-left-neg-1 tablet:margin-right-5">
-          <span class="font-sans-xl text-success-dark text-bold display-block"><%= @submissions_count["completed"] || 0 %></span>
-          <span class="font-sans-lg text-success-dark text-bold">Completed</span>
+      <div class="display-flex flex-row tablet:margin-left-auto tablet:margin-top-0 margin-top-3">
+        <div class="text-center bg-green-cool-vivid-60v padding-1 margin-right-1" style="width: 32%">
+          <span class="font-sans-xl text-white text-bold display-block"><%= @submissions_count["completed"] || 0 %></span>
+          <span class="font-sans-3xs tablet:font-sans-sm text-white">COMPLETED<br>EVALUATIONS</span>
```

Comment:
capitalization can be handled by css property `text-transform`, which is `text-uppercase` in USWDS

Reply:
e.g.
```suggestion
          <span class="font-sans-3xs tablet:font-sans-sm text-white text-uppercase">Completed<br>Evaluations</span>
```